.github/actions/set-up-webserver/action.yml

```diff
@@ -0,0 +1,62 @@
+name: "Set up webserver"
+description: "Set up a webserver with PHP and a database"
+inputs:
+  php-version:
+    description: "PHP version to install"
+    required: true
+  database-version:
+    description: "Database (mysql/mariadb) version to install"
+    required: true
+outputs: {}
+
+runs:
+  using: "composite"
+  steps:
+    - name: Prepare for database
+      shell: bash
```

Comment:
This seems redundant (also below), as "bash" is the default.
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#defaultsrunshell

Reply:
It seems the rules might be different for an action file vs a workflow file. Without specifying the shell, I get a schema validation error: `One of the property sets is required: property 'shell' or property 'uses'`

So in this situation, it seems it's required.

Reply:
Ah, wasn't aware of that difference. Many thanks for clarifying.

Resolved. :+1: